packages/tools/kadena-cli/rollup.config.mjs

```diff
@@ -0,0 +1,21 @@
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import resolve from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
+
+export default {
+  input: 'src/index.ts',
+  output: {
+    dir: './lib',
+    format: 'es',
+  },
```

Comment:
Is this correct? Just a check that it isn’t `esm`

Reply:
https://rollupjs.org/configuration-options/#output-format
It's correct